### PR TITLE
Implement TCPDFEngine

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

- Implement TCPDF engine.
- Removed MPDF.

## How To Test This

1. Set PDF engine in system settings.
2. Test that both PDF engines work as expected (You will only see TCPDF unless you have the mPDF class).

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [X] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

File: lib/PDF/FPDF/TFPDFEngine.php [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        $this->pdf->Write(0, $html);
+    }
+
+    /**
+     * @param string|array $html
+     * @return void
+     */
+    public function writeFooter(string $html): void
+    {
+        $this->pdf->Write(0, $html);
```
Comment (clemente-raposo) [inline lib/PDF/FPDF/TFPDFEngine.php:113]: Hi @Dillon-Brown `writeFooter` and `writeHeader` have the same implementation is that correct?

Comment (Dillon-Brown) [reply lib/PDF/FPDF/TFPDFEngine.php:113]: Yup, instead of defining sections the header is just called first and footer called last.

**Comment by Dillon-Brown** [comment]
Latest push is to fix unit tests.

File: lib/PDF/FPDF/TFPDFEngine.php (hunk 1 above)
Comment (clemente-raposo) [reply lib/PDF/FPDF/TFPDFEngine.php:113]: ok thank you

**Review by clemente-raposo — DISMISSED** [review]
code looks ok

**Review by jack7anderson7 — APPROVED** [review]

**Review by clemente-raposo — APPROVED** [review]
changes look ok

**Comment by Mac-Rae** [comment]
Assessed :+1:

**Review by Mac-Rae — APPROVED** [review]

**Pull request merged**